src/clients/SpokePoolClient.ts

```diff
@@ -675,9 +676,13 @@ export class SpokePoolClient extends BaseAbstractClient {
       });
       this.earlyDeposits = earlyDeposits;
 
-      const dataForQuoteTime: { realizedLpFeePct: BigNumber | undefined; quoteBlock: number }[] = await Promise.all(
-        depositEvents.map((event) => this.computeRealizedLpFeePct(event))
```

Comment:
Can we remove `computeRealizedLpFeePct` function now?

Reply:
Yeah, that would be the intention (or convert it to a wrapper for `batchCompute...()` for a computing a single realizedLpFeePct, as we do when searching for historical deposits).

I've left it in there for the moment because it allowed me to quickly A/B test in my local branch.